python/dask_cudf/dask_cudf/accessors.py

```diff
@@ -2,37 +2,160 @@
 
 
 class ListMethods:
-    # TODO: Maybe use inheritance instead?
     def __init__(self, d_series):
         self.d_series = d_series
 
     def len(self):
+        """
+        Computes the length of each element in the Series/Index.
+        Returns
+        -------
+        Series or Index
+        Examples
```

Comment:
```suggestion
        Series or Index
        
        Examples
```

Similarly, we need newlines between sections below.